src/gt4sd/algorithms/generation/torchdrug/abc.py

```diff
@@ -0,0 +1,4 @@
+import torch
```

Comment:
Do we still need this after the change in the setup? If yes, we should find to find a way to renable OpenMP after the import as quickly discussed.

Reply:
Yes, we still need this and this setup is indeed a way to renable openMP after the torchdrug import. 
We disable openMP in `abc.py` which always runs first in the init. In `core.py` we import the torchdrug things and then afterwards we disable it here again.

Reply:
Same problem like for the sentencepiece issue, if I put the the line to re-enable openmp in the torchdrug file, directly after the last torchdrug import, we get a E402 since we dont have all module imports at the top

Reply:
We could move it below the last import in the `implementation.py` but then I find it cleaner here especially since this init file is always executed even if we start a shell like:
```py
from gt4sd.algorithms.generation.torchdrug.implementation import *
```

Reply:
makes sense 👍

Reply:
Actually we could simply put a check and set it to False, and optionally reenable it, only in case had_openmp is indeed True at the time of import. Otherwise we risk to enable it even if OpenMP is not there.

Reply:
Done in 694b57f